### DISCUSSION TITLE
Support for Linux ARM Manjaro

### DISCUSSION THREAD

**Discussion by girl313** [rustdesk/rustdesk#1384] [General]
Hi hi,

Is there any way to run it on: "Manjaro-ARM-kde-plasma-rpi4-22.08" ?

Or add support for: https://flathub.org/

Thanks.

**Comment by Heap-Hop** [comment]
We havn't built on `Manjaro-ARM-kde-plasma-rpi4-22.08`.

About arm build:
https://github.com/rustdesk/rustdesk/issues/175 
https://github.com/rustdesk/rustdesk/discussions/697

Now we are rewriting UI with Flutter. 
There may be new ways to build on arm in the future.
But low priority (Arm support) for now.
https://github.com/rustdesk/rustdesk/discussions/763

It would be better if you can try to build it on your own device and share it here :) .

**Comment by girl313** [comment]
If I managed to build and put it into install, I wouldn't be writing here...
